### PR TITLE
DEVX-1161: Create cp-all-in-one-community

### PR DESCRIPTION
**Pull request by ybyzek**

### CODE REVIEW

**Comment by rspurgeon** [comment]
@ybyzek I'm getting the following when trying to pull the images for this

`
dc pull
Pulling zookeeper       ... error
Pulling broker          ... error
Pulling schema-registry ... error
Pulling connect         ... error
Pulling ksql-server     ... error
Pulling ksql-cli        ... error
Pulling ksql-datagen    ... done
Pulling rest-proxy      ... done

ERROR: for zookeeper  b'manifest for confluentinc/cp-zookeeper:5.3.1-SNAPSHOT not found: manifest unknown: manifest unknown'

ERROR: for schema-registry  b'manifest for confluentinc/cp-schema-registry:5.3.1-SNAPSHOT not found: manifest unknown: manifest unknown'

ERROR: for broker  b'manifest for confluentinc/cp-kafka:5.3.1-SNAPSHOT not found: manifest unknown: manifest unknown'

ERROR: for connect  b'manifest for cnfldemos/kafka-connect-datagen:0.1.3-5.3.1-SNAPSHOT not found: manifest unknown: manifest unknown'

ERROR: for ksql-server  b'manifest for confluentinc/cp-ksql-server:5.3.1-SNAPSHOT not found: manifest unknown: manifest unknown'

ERROR: for ksql-cli  b'manifest for confluentinc/cp-ksql-cli:5.3.1-SNAPSHOT not found: manifest unknown: manifest unknown'
`

File: cp-all-in-one-community/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+![image](../images/confluent-logo-300-2.png)
+
+# Overview
+
+This [docker-compose.yml](docker-compose.yml) launches just the community services in Confluent Platform and runs them in containers in your local host, enabling you to build your own development environments (see [cp-all-in-one](../cp-all-in-one/docker-compose.yml) for launching all services in Confluent Platform).
+For an example of how to use this Docker setup, refer to the [Confluent Platform quickstart](https://docs.confluent.io/current/quickstart/index.html)
+
+# Additional Examples
+
+For additional examples that showcase streaming applications within an event streaming platform, please refer to [these demos](https://github.com/confluentinc/examples).
```
Comment (JimGalasyn) [inline cp-all-in-one-community/README.md]:
```suggestion
For additional examples that showcase streaming applications within an event streaming platform, see [these demos](https://github.com/confluentinc/examples).
```

File: cp-all-in-one-community/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+![image](../images/confluent-logo-300-2.png)
+
+# Overview
+
+This [docker-compose.yml](docker-compose.yml) launches just the community services in Confluent Platform and runs them in containers in your local host, enabling you to build your own development environments (see [cp-all-in-one](../cp-all-in-one/docker-compose.yml) for launching all services in Confluent Platform).
```
Comment (JimGalasyn) [inline cp-all-in-one-community/README.md]:
```suggestion
This [docker-compose.yml](docker-compose.yml) launches only the community services in Confluent Platform and runs them in containers in your local host, enabling you to build your own development environments (see [cp-all-in-one](../cp-all-in-one/docker-compose.yml) for launching all services in Confluent Platform).
```

File: README.md [hunk 3]
```diff
@@ -77,6 +77,7 @@ We have several resources that launch just the services in Confluent Platform wi
 Using these as a foundation, you can then add any connectors or applications.
 
 * [cp-all-in-one](cp-all-in-one/README.md): This Docker Compose file launches all services in Confluent Platform, and runs them in containers in your local host.
+* [cp-all-in-one-community](cp-all-in-one-community/README.md): This Docker Compose file launches just the community services in Confluent Platform, and runs them in containers in your local host.
```
Comment (JimGalasyn) [inline README.md]:
```suggestion
* [cp-all-in-one-community](cp-all-in-one-community/README.md): This Docker Compose file launches only the community services in Confluent Platform, and runs them in containers in your local host.
```

**Review by JimGalasyn — APPROVED** [review]
LGTM, with a couple of tiny suggestions.

**Review by rspurgeon — APPROVED** [review]
LGTM tested the docker behavior

**Pull request merged**